src/components/Main/index.jsx

```diff
@@ -2,25 +2,31 @@ import React from 'react';
 import PropTypes from 'prop-types';
 import './styles.css';
 
-// -------- Styles --------
+// -------- Inline Styles --------
 const errorStyle = {
   fontSize: '15px',
   color: '#ff0000',
 };
 const isFetchingStyle = {
   height: '18px'
 };
-// ------ End Styles ------
+// ------ End Inline Styles ------
 
 function LocationList(props) {
   const { data } = props;
-  const listItems = data.map(item =>
+  const listItems = data.slice(0,5).map(item =>
```

Comment:
the idea is to save in local storage only the last 5 places, so you wouldn't need to slice you data, and you wouldn't be saving extra places.

Reply:
Initially i did that. But as the doc states: "display the last 5" and not "store or save the last 5" I changed it xD.
But I revert that of course.

Reply:
Yes, do it, now you can save infinite number of places if I'm not wrong